infrastructure/batch/job_definition.tf

```
@@ -0,0 +1,37 @@
+resource "aws_batch_job_definition" "scpca_portal_project" {
+  name = "scpca-portal-project-job-definition-${var.user}-${var.stage}"
+  type = "container"
+
+  platform_capabilities = [
+    "FARGATE",
+  ]
+  container_properties = jsonencode({
+    # dispatch_to_batch command will override the below command during job submission, with
+    # cotaninerOverrides={"command": ["sportal", "generate-computed-files", "scpca-project-id", "<id>"]}
+    command = ["sportal", "generate-computed-files"]
```

```suggestion
    # command definition expected in cotaninerOverrides when using this job definition
    command = []
```
This file doesn't need to know about how we submit batch jobs as that is subject to change.